data/1.1/_index.md

```diff
@@ -0,0 +1,79 @@
+<!-- Template for the root page of a specification release -->
+<!-- The "date:" field is used as a "publish" date for the automated Hugo processing -->
+---
+title: "Jakarta Data 1.1 (under development)"
+date: 2025-01-28
+summary: "Enhancements to the Jakarta Data spec"
+---
+
+### New features, enhancements or additions
+* Ability to define query restrictions dynamically (https://github.com/jakartaee/data/issues/829)
+* Life cycle events (https://github.com/jakartaee/data/issues/373)
+* Bring parameter based automatic query methods up to the level of what Query by Method Name can do for static queries (https://github.com/jakartaee/data/issues/857)
+* Query Fragments (https://github.com/jakartaee/data/issues/546)
+* Multiple item select lists/DTOs/Entity Views (https://github.com/jakartaee/data/issues/539)
+* Consider additional annotations related to Jakarta Persistence (https://github.com/jakartaee/data/issues/470)
+* Reactive patterns (#17) and integration with Jakarta Concurrency Asynchronous (https://github.com/jakartaee/data/issues/19)
+* Move Jakarta Data Query Language to Jakarta Query specification (if accepted and included in Jakarta EE 12) where it will remain fully compatible.
+* Configuration via Jakarta Config (if Jakarta Config is released and makes it into Jakarta EE 12 in time)
+* Explicitly state where NoSQL databases can raise errors for function the database type is not capable of (https://github.com/jakartaee/data/issues/782)
+* TCK refactoring and enhancements to more logically split out tests that are targeted at various levels of database support (key-value, column, document, graph, relational).
+
+
+### Removals, deprecations or backwards incompatible changes
+* N/A
+
+### Minimum Java SE Version
+<!-- Specify the minimum required Java SE version for this specification -->
+**21**
+
+# Details
+
+* [Jakarta Data 1.1 Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.1)
+* [Jakarta Data 1.1 Specification Document](./jakarta-data-1.1.pdf) (PDF) (Does not exist yet)
+* [Jakarta Data 1.1 Specification Document](./jakarta-data-spec-1.1.html) (HTML) (Does not exist yet)
+* [Jakarta Data 1.1 Javadoc](./apidocs) (Does not exist yet)
+* [Jakarta Data 1.1 TCK](http://downloads.eclipse.org/jakarta/data/1.1/data-tck-1.1.0.zip)
+* Maven coordinates
+  * [jakarta.data:jakarta.data-api:jar:1.1.0](https://search.maven.org/artifact/jakarta.data/jakarta.data-api/1.1.0/jar) (Does not exist yet)
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * [To Be Determined](https://github.com/)
+
+# Compatible Implementations
+
+* [To Be Determined](https://github.com/)
+
+# Ballots
+
+## Release Review
+
+The Release Review Specification Committee Ballot concluded successfully on YYYY-MM-DD with the following results.
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
```

Comment:
The table is not accurate. Use the below content for both review tables.

Representative 	Representative for: 	Vote
Kenji Kazumura 	Fujitsu 	
Emily Jiang, Tom Watson 	IBM 	
Ed Bratt, Dmitry Kornilov 	Oracle 	
Andrew Pielage, Petr Aubrecht 	Payara 	
David Blevins, Jean-Louis Monteiro 	Tomitribe 	
Ivar Grimstad 	EE4J PMC 	
Marcelo Ancelmo, Abraham Marin-Perez 	Participant Members 	
Werner Keil 	Committer Members 	
Jun Qian 	Enterprise Members 	
Zhai Luchao 	Enterprise Members

Reply:
The other option is not to include the table. Please see [this](https://github.com/jakartaee/specifications/pull/800) PR for an example. It is okay not to include the details so that the spec team will record the latest WG info.

Reply:
Thanks for providing the correct list of voters. I updated it in commit https://github.com/jakartaee/specifications/pull/795/commits/d5049a655338307485a6958dad9353f866f8e6d2